tests/security/selinux/selinux_setup.pm

```diff
@@ -17,14 +17,15 @@
 # Maintainer: llzhao <llzhao@suse.com>
 # Tags: poo#40358
 
-use base "consoletest";
+use base 'opensusebasetest';
 use strict;
 use testapi;
 use utils;
 use version_utils qw(is_sle is_leap);
 
 sub run {
-    select_console "root-console";
+    my ($self) = @_;
+    $self->select_serial_terminal;
```

Comment:
do you have an idea how we could make this look like

```perl
select_console 'serial-console';
```

instead?

Reply:
taking a look into code inside `select_serial_terminal` I would say it is possible only with doing further `select_console` calls inside `select_console` which not look like as a good approach

Also I would like to keep this logic in test code so we are more flexible and faster with changing it in comparison moving it to backend

Reply:
Maybe `$self->select_serial_terminal` it's not clear enough (feel free to rename it). I just wanted to have single function which handles backend logic, which is otherwise in many tests.

Reply:
@pevik as I understand @okurz is actually asking to basically move it to backend . @okurz or I missunderstood you ?

Reply:
@asmorodskyi I was thinking about it as well, when I was doing the original patch, but some parts was in our opensuse tests (`S390_ZKVM`, etc.).